code/game/vehicles/spacepods/equipment.dm

```diff
@@ -186,3 +187,41 @@
 /obj/item/device/spacepod_equipment/cargo/ore/removed(var/mob/user)
 	. = ..()
 	unload()
+
+/obj/item/device/spacepod_equipment/lock
+	name = "pod lock"
+	desc = "You shouldn't be seeing this"
+	icon = 'icons/goonstation/pods/ship.dmi'
+	icon_state = "blank"
+	var/mode = 0
+	var/id = null
+
+// Key and Tumbler System
+/obj/item/device/spacepod_equipment/lock/keyed
+	name = "\improper spacepod tumbler lock"
+	desc = "A locking system to stop podjacking. This version uses a standalone key."
+	icon_state = "pod_locator"
+
+/obj/item/device/spacepod_equipment/lock/keyed/New()
+	..()
+	id = rand(1, 99999)
```

Comment:
bad, rare chance the numbers will clash.

add
```
var/static/id_source = 0
```
to the definition of ```/obj/item/device/spacepod_equipment/lock/keyed```
and then replace this line with ```id = ++id_source```

guarantees unique numerical keys.